src/bindings/python/fluxacct/accounting/job_usage_calculation.py

```diff
@@ -110,7 +89,7 @@ def apply_decay_factor(decay, acct_conn, user=None, bank=None):
     usg_past = []
     usg_past_decay = []
 
-    usg_past = fetch_usg_bins(acct_conn, user, bank)
+    usg_past = usage_factors
```

Comment:
It looks like `usg_past` is defined and then initialized to something else, could you just remove the `usg_past` variable and use `usage_factors` directly instead?